### DISCUSSION TITLE
Trialing Discord for community updates & chats

### DISCUSSION THREAD

**Discussion by kitten** [urql-graphql/urql#3107] [Announcements]
### tl;dr: Trialing Discord communities

**We're trialing Discord communities and channels for updates, coordination, and chats!**
<a href="https://urql.dev/discord">
  <img alt="Discord" src="https://img.shields.io/discord/1082378892523864074?color=7389D8&label&logo=discord&logoColor=ffffff" />
</a>

We'll be trialing Discord channels to announce updates to the project for which release notes, issues, and discussions aren't enough.
We're also hoping that this will ease the way in which we can support you in answering questions, and encourage you — our community — to get in touch with each other and help each other as well.
Initially, we're thinking of splitting this into three channels:

- `#urql-discussions`: A general chat channel to chat about `urql`, proposals, PRs, etc
- `#urql-updates`: A channel where we'll announce releases, important changes, and ask for feedback
- `#urql-help`: A forum help channel, which is a more interactive alternative to our GH Discussions

These channels are hosted on the @0no-co Discord community, which also will include corresponding channels for [wonka](https://github.com/0no-co/wonka), a project that `urql` of course builds on and uses.

Let us know if you think this is helpful, check out our Discord, and feel free to ask questions there! (we have some guidance on Discord, e.g. using threads, etc)

**https://urql.dev/discord**
